app/src/androidTest/java/com/android/unio/end2end/EndToEndTest.kt

```diff
@@ -78,9 +100,13 @@ open class EndToEndTest : FirebaseEmulatorFunctions {
       Firebase.firestore.useEmulator(HOST, Firestore.PORT)
       Firebase.auth.useEmulator(HOST, Auth.PORT)
       Firebase.functions.useEmulator(HOST, Functions.PORT)
-      println("Firebase emulators configured successfully.")
     } catch (e: IllegalStateException) {
-      // Firebase emulators have already been initialized. Skipping configuration.
+      Log.d("EndToEndTest", "Firebase Emulators are already in use. $e")
```

Comment:
Maybe change this Log.d to a Log.e?

Reply:
It should maybe be a warning or an info log